src/doc_scan_service/session/create/session.specification.builder.js

```diff
@@ -154,6 +154,12 @@ class SessionSpecificationBuilder {
     return this;
   }
 
+  withIdentityProfile(identityProfile) {
+    Validation.isPlainObject(identityProfile, 'identityProfileConfig');
```

Comment:
Should be 
```
Validation.isPlainObject(identityProfileRequirements, 'identityProfileRequirements')
```
The first param is what you want to validate, the second one is the name of the fields so if an error is raised, it will refer to that field.